index.js

```diff
@@ -74,6 +74,27 @@ var prepareContext = function(ctx) {
   }
 
   /**
+   * Remove the bogus context from any standalone sassdoc comments.
+   * (We detect these if the context starts more than one line after the
+   * sassdoc comment ends.)
+   */
+  ctx.data.forEach(function(item) {
+    if (!item.context || !item.context.line) {
+      return;
+    }
+    if (item.context.type === 'unknown') {
+      item.context.type = 'prose';
+      item.context.line.end = item.context.line.start;
+    }
+    if (item.context.line.start > item.commentRange.end + 1) {
+      item.context = {
+        type: 'prose',
+        line: item.commentRange
+      };
+    }
```

Comment:
Hmm... The case I'm trying to detect here is where there are blank lines in between a sassdoc comment and a selector (because in that case the comment is probably meant to be prose, but the scss-comment-parser associates it with the selector). Maybe we can actually get the contents of line number `commentRange.end + 1` and see if it's blank.